.test-infra/jenkins/job_LoadTests_ParDo_Flink_Python.groovy

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import CommonTestProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+import Flink
+import Docker
+
+String now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def scenarios = { datasetName, sdkHarnessImageTag -> [
+        [
+                title        : 'ParDo Python Load test: 2GB 100 byte records 10 times',
+                itClass      : 'apache_beam.testing.load_tests.pardo_test:ParDoTest.testParDo',
+                runner       : CommonTestProperties.Runner.PORTABLE,
+                jobProperties: [
+                        job_name             : 'load-tests-python-flink-batch-pardo-1-' + now,
+                        project              : 'apache-beam-testing',
+                        publish_to_big_query : false,
+                        metrics_dataset      : datasetName,
+                        metrics_table        : 'python_flink_batch_pardo_1',
+                        input_options        : '\'{' +
+                                '"num_records": 20000000,' +
+                                '"key_size": 10,' +
+                                '"value_size": 90}\'',
+                        iterations           : 10,
+                        number_of_counter_operations: 0,
+                        number_of_counters   : 0,
+                        parallelism          : 5,
+                        job_endpoint         : 'localhost:8099',
+                        environment_config   : sdkHarnessImageTag,
+                        environment_type     : 'DOCKER',
+                ]
+        ],
+        [
+                title        : 'ParDo Python Load test: 2GB 100 byte records 200 times',
+                itClass      : 'apache_beam.testing.load_tests.pardo_test:ParDoTest.testParDo',
+                runner       : CommonTestProperties.Runner.PORTABLE,
+                jobProperties: [
+                        job_name             : 'load-tests-python-flink-batch-pardo-2-' + now,
```

Comment:
Tiny nit, but I think you can do it even more concise: 

```
"load-tests-python-flink-batch-pardo-2-${now}"
```

wdyt?

Reply:
For me, appending string suffixes via '+' operator is a tiny bit more cleaner. I don't think we need to replace this with the ${} syntax.